openedx/features/course_experience/tests/views/test_course_home.py

```diff
@@ -150,7 +173,7 @@ def setUp(self):
         CourseUserType.UNENROLLED_STAFF,
     )
     def test_home_page(self, user_type):
-        self.user = self.create_user_for_course(self.course, user_type)
+        self._create_test_user(user_type)
```

Comment:
This also got accidentally picked up in the rebase. The original is the correct logic.